includes/editor-templates/panel-elements.php

```diff
@@ -12,25 +12,45 @@
 	</div>
 	<div id="elementor-panel-elements-search-area"></div>
 	<div id="elementor-panel-elements-notice-area"></div>
-	<div id="elementor-panel-elements-wrapper"></div>
+	<div id="elementor-panel-elements-wrapper"
+<!--		 style="flex: 1;-->
+<!--		display: flex;-->
+<!--		flex-direction: column;"-->
+	></div>
 </script>
 
 <script type="text/template" id="tmpl-elementor-panel-categories">
 	<div id="elementor-panel-categories"></div>
-
-	<?php
-	$get_pro_details = apply_filters( 'elementor/editor/panel/get_pro_details', [
-		'link' => 'https://go.elementor.com/pro-widgets/',
-		'message' => __( 'Get more with Elementor Pro', 'elementor' ),
-		'button_text' => __( 'Upgrade Now', 'elementor' ),
-	] );
-	?>
+			<?php
```

Comment:
Do have to make changes here? I would try to keep the number of changes as low as possible, to make the review process easier.